pytorch_blade/tests/mhlo/layer_norm.mlir

```diff
@@ -1,418 +0,0 @@
-// RUN: torch-mlir-opt <%s --torch-backend-to-mhlo-backend-pipeline -split-input-file -verify-diagnostics | FileCheck %s
```

Comment:
why delete this ut ?

Reply:
I agree with the recommendations in https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing, that's those "macro expansion" is not useful.

Reply:
after the offline discussion:
1. find a way to reduce the maintenance cost, e.t.  using a wildcard to check some key points.
2. only check the code that maintains in the BladeDISC repo.
3. remove some filecheck ut that maintains in the torch-mlir repo.